### ISSUE TITLE
Create a Read-Only role for Collection Grants

### ISSUE THREAD

**Issue by cd-rite** [NUWCDIVNPT/stig-manager#322]
Users have expressed desire to provide read-only access to Collections. Would provide visibility at the "Full" level, without ability to modify reviews. 

Option: Should this be a modifier applied to an existing Grant or a new type of Grant?

Question: Could/should a Restricted user have Grants in a collection that allows them to modify specific STIG/Asset assignments, but overall Read-Only access to a collection?

Question: Does/should the Client check that the user token includes the scopes it requested?

Note: 
Overall Read-Only access for the whole app can already be accomplished by assigning appropriate Scopes (ie. just the ``:read`` ones) .

**Comment by dtester1** [comment]
Could the restricted access grant be used with some additional options in manage user access to implement this?
Seems like the interface is there to support something like this.  Restricted Read Access is what I am currently thinking.

**Comment by csmig** [comment]
Yes. I'll post a followup comment with the initial design specification we are developing against. It supports User Groups and a configurable Review ACL for all roles, not only Restricted. The ACL rules will support an `access` property whose value is from the set `read, read/write`. The design is still subject to minor changes.

**Comment by cd-rite** [comment]
resolved: #1487
